app.json

```diff
@@ -26,7 +24,12 @@
       }
     },
     "web": {
-      "favicon": "./assets/favicon.png"
+      "favicon": "./assets/favicon.png",
+      "build": {
+        "babel": {
+          "include": ["galio-framework"]
```

Comment:
Why did you add babel here? Please remove this.

Reply:
It's because we added 'galio framework' package and thought it would work but it did not. We will remove it.

Reply:
It has been removed